src/main/scala/smallpt.scala

```diff
@@ -1,208 +0,0 @@
-package demo
```

Comment:
If this is a skeleton, it makes more sense to have a simple hello world. This is still available in the scala-native repository.